packages/venia-ui/lib/components/RichContent/PageBuilder/missing.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import defaultClasses from './missing.css';
+import { mergeClasses } from '../../../classify';
+
+const Missing = ({ classes, contentType }) => {
+    classes = mergeClasses(defaultClasses, classes);
+    return (
+        <div className={classes.missing}>
+            <strong>Error:</strong> No component for{' '}
```

Comment:
Consider wrapping bare React text nodes in quotes or backticks. As shown here, syntax highlighting has trouble with them, and the quotes will help us find stuff later when we want to deal with translation.